apps/condo/domains/user/schema/User.js

```diff
@@ -162,13 +162,13 @@ const User = new GQLListSchema('User', {
         },
 
         importRemoteSystem: {
-            schemaDoc: 'External provider for users',
```

Comment:
do you want to remove this fields?

Reply:
I'll do that later, after data migration to be completed. Migration is done by separate script